output/crafting-mementos.json

```diff
@@ -1,8 +1,9 @@
 {
-  "crucible": [18179099, 1429874803, 3617209212],
```

Comment:
Feels like crucible should still be a thing?

Reply:
Crucible mementos don't exist, Bungie just had these defs in early under a different name and now renamed them IIUC.

Reply:
Ah, I see. I don't really get mementos.